config/webpack/webpack.config.dev.js

```diff
@@ -9,16 +8,14 @@ var LodashModuleReplacementPlugin = require("lodash-webpack-plugin");
 config.output.filename = config.output.filename.replace(/\.min\.js$/, ".js");
 config.output.pathinfo = true;
```

Comment:
Somewhat related, I think we can get rid of this line too. The modern webpack dev output is of the form

```js
{
"./index.js":
"../../node_modules/foo/flie.js":
}
```

... which is what we really wanted pathinfo for anyways